packages/create-toolpad-app/tests/index.spec.ts

```diff
@@ -93,6 +93,59 @@ test(
   TEST_TIMEOUT,
 );
 
+test(
+  'create-toolpad-app can bootstrap a Toolpad Core app',
+  async () => {
+    testDir = await fs.mkdtemp(path.resolve(os.tmpdir(), './test-app-'));
+    cp = execa(cliPath, [testDir], {
+      cwd: currentDirectory,
+    });
+    cp.stdout?.pipe(process.stdout);
+    cp.stderr?.pipe(process.stderr);
+    const result = await cp;
+    expect(result.stdout).toMatch('Run the following to get started');
+    const packageJsonContent = await fs.readFile(path.resolve(testDir, './package.json'), {
+      encoding: 'utf-8',
+    });
+    const packageJson = JSON.parse(packageJsonContent);
+    expect(packageJson).toEqual(
+      expect.objectContaining({
+        dependencies: expect.objectContaining({
+          '@toolpad/core': expect.any(String),
+        }),
+        scripts: expect.objectContaining({
+          dev: 'next dev',
+          build: 'next build',
+          start: 'next start',
+          lint: 'next lint',
+        }),
+      }),
+    );
+
+    // check that file exists or not in the directory
+    const gitignore = await fs.readFile(path.resolve(testDir, './.gitignore'), {
```

Comment:
I guess you can also use `fs.exists` or `fs.stat` instead of `readFile` here (not sure which is better or the best way nowadays, would have to try it)

Reply:
Node.js deprecated `fs.exists` in favour of `fs.access` and `fs.stat`, both of which warn against using them to check for existence before attempting to read or write the file, for race condition worries: https://nodejs.org/api/fs.html#fsstatpath-options-callback

Reply:
Oh ok great I didn't know (wasn't sure like I said :D), thanks for clarifying!